### PR TITLE
Update our TLS configuration docs

### PR DESCRIPTION
**Pull request by tas50**
Suggest better defaults on 13.x and document the new defaults in the 14.3 release.

### CODE REVIEW

File: docs-chef-io/content/server/v13_2/config_rb_server_optional_settings.md [hunk 1]
```diff
@@ -742,11 +742,14 @@ This configuration file has the following settings for `nginx`:
...
+    releases 10.16.4 and later on Linux/Unix/macOS and 12.8 or later on
+    Windows. If it is necessary to support these older end-of-life
+    Chef Infra Client releases this value can be set to `'TLSv1.1 TLSv1.2'`.
 
     ```ruby
-    nginx['ssl_protocols'] = 'TLSv1 TLSv1.1 TLSv1.2'
+    nginx['ssl_protocols'] = 'TLSv1.2'
     ```
 
     Default value: `TLSv1 TLSv1.1 TLSv1.2`.
```
Comment (teknofire) [inline docs-chef-io/content/server/v13_2/config_rb_server_optional_settings.md:755]: This should be changed to `TLSv1.2` as well

File: docs-chef-io/content/server/v13_2/config_rb_server_optional_settings.md [hunk 2]
```diff
@@ -742,11 +742,14 @@ This configuration file has the following settings for `nginx`:
 
 `nginx['ssl_protocols']`
 
-:   The SSL protocol versions that are enabled. For the highest possible
-    security, disable SSL 3.0 and allow only TLS:
+:   The SSL protocol versions that are enabled.For enhanced security set
+    this value to `'TLSv1.2'`. TLS 1.2 is supported on Chef Infra Client
+    releases 10.16.4 and later on Linux/Unix/macOS and 12.8 or later on
+    Windows. If it is necessary to support these older end-of-life
+    Chef Infra Client releases this value can be set to `'TLSv1.1 TLSv1.2'`.
```
Comment (IanMadd) [inline docs-chef-io/content/server/v13_2/config_rb_server_optional_settings.md:749]:
```suggestion
:   The SSL protocol versions that are enabled. For enhanced security set
    this value to `'TLSv1.2'`. TLS 1.2 is supported on Chef Infra Client 10.16.4 
    and later on Linux, Unix, and macOS, and on Chef Infra Client 12.8 and later on
    Windows. If it is necessary to support these older end-of-life
    Chef Infra Client releases, set this value to `'TLSv1.1 TLSv1.2'`.
```

File: docs-chef-io/content/server/v13_2/reusable_text/server_tuning_nginx.md [hunk 3]
```diff
@@ -27,18 +27,14 @@ Infra Server to use SSL certificates:
...
-    authenticated version negotiation between Chef Infra Client and Chef
-    Infra Server, which ensures the latest version of the TLS protocol
-    is used. For the highest possible security, it is recommended to
-    disable SSL 3.0 and allow all versions of the TLS protocol. For
-    example:
+:   The SSL protocol versions that are enabled for the Chef Infra Server API.
+    For enhanced security set this value to `'TLSv1.2'`. TLS 1.2 is supported on
+    Chef Infra Client releases 10.16.4 and later on Linux/Unix/macOS and 12.8
+    or later on Windows. If it is necessary to support these older end-of-life
+    Chef Infra Client releases this value can be set to `'TLSv1.1 TLSv1.2'`.
```
Comment (IanMadd) [inline docs-chef-io/content/server/v13_2/reusable_text/server_tuning_nginx.md:34]:
```suggestion
:   The SSL protocol versions that are enabled for the Chef Infra Server API. 
    For enhanced security set this value to `'TLSv1.2'`. TLS 1.2 is supported on 
    Chef Infra Client 10.16.4 and later on Linux, Unix, and macOS, and on Chef 
    Infra Client 12.8 and later on Windows. If it is necessary to support these 
    older end-of-life Chef Infra Client releases, set this value to `'TLSv1.1 TLSv1.2'`.
```

File: docs-chef-io/content/server/v14/config_rb_server_optional_settings.md [hunk 4]
```diff
@@ -720,14 +720,19 @@ This configuration file has the following settings for `nginx`:
...
 
-:   The SSL protocol versions that are enabled. For the highest possible
-    security, disable SSL 3.0 and allow only TLS:
+:   The SSL protocol versions that are enabled for the Chef Infra Server API.
+    Starting with Chef Infra Server 14.3 this value defaults to `'TLSv1.2'` for
+    enhanced security. Previous releases defaulted to `'TLSv1 TLSv1.1 TLSv1.2'`,
+    which allowed for less secure SSL connections. TLS 1.2 is supported on
+    Chef Infra Client releases 10.16.4 and later on Linux/Unix/macOS and 12.8
+    or later on Windows. If it is necessary to support these older end-of-life
+    Chef Infra Client releases this value can be set to `'TLSv1.1 TLSv1.2'`.
```
Comment (IanMadd) [inline docs-chef-io/content/server/v14/config_rb_server_optional_settings.md:729]:
```suggestion
:   The SSL protocol versions that are enabled for the Chef Infra Server API.
    Starting with Chef Infra Server 14.3, this value defaults to `'TLSv1.2'` for
    enhanced security. Previous releases defaulted to `'TLSv1 TLSv1.1 TLSv1.2'`,
    which allowed for less secure SSL connections. TLS 1.2 is supported on
    Chef Infra Client 10.16.4 and later on Linux, Unix, and macOS, and on Chef 
    Infra Client 12.8 and later on Windows. If it is necessary to support these older end-of-life
    Chef Infra Client releases, set this value to `'TLSv1.1 TLSv1.2'`.
```

File: docs-chef-io/content/server/v14/reusable_text/server_tuning_nginx.md [hunk 5]
```diff
@@ -27,18 +27,16 @@ Infra Server to use SSL certificates:
...
-    is used. For the highest possible security, it is recommended to
-    disable SSL 3.0 and allow all versions of the TLS protocol. For
-    example:
+:   The SSL protocol versions that are enabled for the Chef Infra Server API.
+    Starting with Chef Infra Server 14.3 this value defaults to `'TLSv1.2'` for
+    enhanced security. Previous releases defaulted to `'TLSv1 TLSv1.1 TLSv1.2'`,
+    which allowed for less secure SSL connections. TLS 1.2 is supported on
+    Chef Infra Client releases 10.16.4 and later on Linux/Unix/macOS and 12.8
+    or later on Windows. If it is necessary to support these older end-of-life
+    Chef Infra Client releases this value can be set to `'TLSv1.1 TLSv1.2'`.
```
Comment (IanMadd) [inline docs-chef-io/content/server/v14/reusable_text/server_tuning_nginx.md:36]:
```suggestion
:   The SSL protocol versions that are enabled for the Chef Infra Server API.
    Starting with Chef Infra Server 14.3, this value defaults to `'TLSv1.2'` for
    enhanced security. Previous releases defaulted to `'TLSv1 TLSv1.1 TLSv1.2'`,
    which allowed for less secure SSL connections. TLS 1.2 is supported on
    Chef Infra Client 10.16.4 and later on Linux, Unix, and macOS, and on Chef 
    Infra Client 12.8 and later on Windows. If it is necessary to support these older end-of-life
    Chef Infra Client releases, set this value to `'TLSv1.1 TLSv1.2'`.
```

File: docs-chef-io/content/server/v13_2/config_rb_server_optional_settings.md (hunk 1 above)
Comment (tas50) [reply docs-chef-io/content/server/v13_2/config_rb_server_optional_settings.md:755]: These are the 13.2 docs so they still default to the old value

**Pull request merged**